Themes/default/css/index.css

```diff
@@ -57,6 +58,10 @@ textarea, textarea.editor {
 input, button, select {
 	font-size: 0.9em;
 }
+/* Height fixation */
+input, select {
```

Comment:
Won't that break select lists? See the avatar selection...

`select:not([size])`

On Mon, Mar 28, 2016 at 6:32 PM, Mert Alınbay <notifications@github.com>
wrote:

> In Themes/default/css/index.css
> <https://github.com/SimpleMachines/SMF2.1/pull/3348#discussion_r57661574>:
>
> > @@ -57,6 +58,10 @@ textarea, textarea.editor {
> >  input, button, select {
> >  	font-size: 0.9em;
> >  }
> > +/* Height fixation */
> > +input, select {
>
> because they have different heights ? and different browsers natively
> handle height/width differently. We need a standart on that.
>
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
> <https://github.com/SimpleMachines/SMF2.1/pull/3348/files/2a16456c03021c3dcede045f4abc81c63d46233c#r57661574>
>